WooCommerce/WooCommerceTests/ViewRelated/Products/Edit Product/Subscription/SubscriptionTrialViewModelTests.swift

```diff
@@ -0,0 +1,161 @@
+import XCTest
+import Yosemite
+@testable import WooCommerce
+import WooFoundation
+
+final class SubscriptionTrialViewModelTests: XCTestCase {
+    private let samplePeriod: SubscriptionPeriod = .month
+
+    // MARK: `trialPeriodDescription`
+
+    func test_trialPeriodDescription_returns_expected_description_for_singular_trialPeriod_interval() {
+        // Given
+        let subscription = ProductSubscription.fake().copy(trialLength: "1", trialPeriod: samplePeriod)
+        let viewModel = SubscriptionTrialViewModel(subscription: subscription) { _, _, _ in }
+
+        // Then
+        XCTAssertEqual(viewModel.trialPeriodDescription, "month")
```

Comment:
Same comment above about testing localized strings.

Reply:
Fixed in https://github.com/woocommerce/woocommerce-ios/pull/11173/commits/9c5027d813e80a7386ab941a9a4e50179627afdd